modules/oc-mirror-oci-format.adoc

```diff
@@ -20,47 +33,57 @@ include::snippets/technology-preview.adoc[]
+    graph: false
+  operators:
+  - catalog: oci:///home/user/oc-mirror/my-oci-catalog                       <3>
+    packages:
+    - name: aws-load-balancer-operator
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.13           <4>
+    packages:
+    - name: rhacs-operator
+  additionalImages:
+  - name: registry.redhat.io/ubi8/ubi:latest                                 <5>
 ----
-+
-[NOTE]
-====
-When using the OCI feature, only the `mirror.operators.catalog` setting is available for use.
-
-The `storageConfig` setting is ignored in favor of the location passed in to the `oc mirror` command.
-====
+<1> Set the back-end location to save the image set metadata to. This location can be a registry or local directory. It is required to specify `storageConfig` values.
+<2> Optionally, include an {product-title} release to mirror from `registry.redhat.io`.
+<3> Specify the absolute path to the location of the OCI catalog on disk. The path must start with `oci://` when using the OCI feature.
+<4> Optionally, specify additional Operator catalogs to pull from a registry.
+<5> Optionally, specify additional images to pull from a registry.
 
-.. Run the `oc mirror` command to mirror the images from the specified image set configuration to disk:
+. Run the `oc mirror` command to mirror the OCI catalog to a target mirror registry:
 +
 [source,terminal]
 ----
 $ oc mirror --config=./imageset-config.yaml \ <1>
-  --use-oci-feature \                         <2>
-  --oci-feature-action=copy \                 <3>
-  oci://my-oci-catalog                        <4>
+  --include-local-oci-catalogs                <2>
+  docker://registry.example:5000              <3>
 ----
 <1> Pass in the image set configuration file. This procedure assumes that it is named `imageset-config.yaml`.
-<2> Use the `--use-oci-feature` flag to enable the OCI feature.
-<3> To copy the catalog to disk, set the `--oci-feature-action` flag to `copy`.
-<4> Specify a directory on disk where you want to output the catalog. This procedure assumes that it is named `my-oci-catalog`. The path must start with `oci://`. If the specified directory is not a full path, the directory is created in the current working directory where the `oc mirror` command is run.
+<2> Use the `--include-local-oci-catalogs` flag to enable the OCI feature.
+<3> Specify the registry to mirror the image set file to. The registry must start with `docker://`. If you specify a top-level namespace for the mirror registry, you must also use this same namespace on subsequent executions.
++
+Optionally, you can specify other flags to adjust the behavior of the OCI feature:
++
+`--oci-insecure-signature-policy`:: Do not push signatures to the target mirror registry.
++
+`--oci-registries-config`:: Specify the path to a TOML-formatted `registries.conf` file. You can use this to mirror from a different registry, such as a pre-production location for testing, without having to change the image set configuration file.
```

Comment:
I'd add this to be on the safe side: 
This only applies to the local oci catalogs, and not to the rest of the mirrored content.